ocaml/xapi/record_util.ml

```diff
@@ -198,6 +198,25 @@ let string_to_network_default_locking_mode = function
   | "disabled" -> `disabled
   | s -> raise (Record_failure ("Expected 'unlocked' or 'disabled', got "^s))
 
+let network_purpose_to_string = function
```

Comment:
To ensure that the match is complete, we should add the type: `..._to_string : API.network_purpose -> string = `

Reply:
Good thought.